File: UI/src/main/java/ca/warp7/rt/feature/vc/VCFeature.java
package ca.warp7.rt.feature.vc;

import ca.warp7.rt.core.feature.Feature;
import ca.warp7.rt.core.feature.FeatureItemTab;
import javafx.collections.FXCollections;

import java.util.List;

public class VCFeature implements Feature {

    @Override
    public List<FeatureItemTab> getLoadedTabs() {
        return FXCollections.singletonObservableList(new FeatureItemTab(
                "Verification Center", "fas-check:16:gray", getFeatureId(),
                FeatureItemTab.Group.SingleTab, ""
        ));
    }

    @Override
    public String getFeatureId() {
        return "vc";
    }
}

File: UI/src/main/kotlin/ca/warp7/rt/feature/vc/VCFeature.kt
package ca.warp7.rt.feature.vc

import ca.warp7.rt.core.feature.Feature
import ca.warp7.rt.core.feature.FeatureItemTab

class VCFeature : Feature {

    override fun getLoadedTabs(): List<FeatureItemTab> {
        return listOf(FeatureItemTab(
                "Verification Center", "fas-check:16:gray", featureId,
                FeatureItemTab.Group.SingleTab, ""
        ))
    }

    override fun getFeatureId(): String {
        return "vc"
    }
}
